components/tabs/Tab.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from "react";
+import { TabListItem } from "./Tabs.styled";
+
+export interface TabProps {
+  activeTab: string;
+  "data-label": string;
```

Comment:
is this like an html standard thing? its very awkwardly named

Reply:
Not sure familiar with this either, could you elaborate a bit on what does it do and how to use it?

Reply:
This is an html standard.

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

As we want to apply this prop *directly * to a div, this is how we do it. TypeScript throws an error without the data-* naming convention, as it's not an intrinsic element.